SpatialGDK/Documentation/content/generating-a-snapshot.md

```diff
@@ -1,37 +1,53 @@
+
+To ensure these Actors are listed in the snapshot, tag them as `[SpatialType]({{urlRoot}}/content/spatial-type)`.
 
 ### Critical entities
 
-Critical entities are entities which are used for functionality critical to the GDK and are never deleted. They are saved into the initial snapshot and must always exist when launching a deployment.
+Critical entities are listed in snapshots by default.
 
-Currently the critical entities are:
+Critical entities are functionality critical for the GDK; do not delete them. SpatialOS needs them for launching a deployment. You save these into your initial snapshot when you generate it. 
 
-* `SpatialSpawner` - an entity with the `PlayerSpawner` component which has a command. Connecting client-workers use this entity to spawn their player.
-* `GlobalStateManager` - an entity with the `GlobalStateManager` component which has a map of singleton classes to entity IDs. This entity is used for orchestrating the replication of [Singleton Actors]({{urlRoot}}/content/singleton-actors.md).
+The critical entities are:
 
-### Placeholders
+* `SpatialSpawner` - an entity with the `PlayerSpawner` component which contains the `spawn_player` command. [Client-workers]({{urlRoot}}/content/glossary#workers) connecting to a [deployment]({{urlRoot}}/content/glossary#deployment) use this entity to spawn their player.
+* `GlobalStateManager` - an entity with the `GlobalStateManager` component which has a map of [singleton]({{urlRoot}}/content/singleton-actors.md) classes to entity IDs (see [Global State manager]({{urlRoot}}/content/glossary#global-state-manager) glossary entry). The GDK uses this entity to orchestrate the replication of [Singleton Actors]({{urlRoot}}/content/singleton-actors.md).
 
-These entities exists only to set up server-worker boundaries in a way that is easy to test in a scenario with two server-workers. They will not spawn as actors when checked out and serve no purpose within the GDK. For most intents and purposes, you can safely ignore them.
+### Placeholder entities
+Placeholder entities are listed in snapshots by default.
 
-## Generating a snapshot
+These entities exists only to set up server-worker boundaries in a way that is easy to test with multiple server-workers. These entities do not spawn as Actors when [checked out]({{urlRoot}}/content/glossary/check-out] by a worker and serve no purpose within the GDK. For most intents and purposes, you can safely ignore them.
 
-To generate a snapshot, use the **Snapshot** button on the SpatialOS GDK for Unreal toolbar in the Unreal Editor:
+## How to generate a snapshot
```

Comment:
Shouldn't we be including the information on how to change the default snapshot save path and such?